docs/usage.md

```diff
@@ -149,4 +154,4 @@ try {
 }
 ```
 
-See the [API documentation](api) for more examples and full API reference information.
+See the [API documentation](/docs/tools/generator/api) for more examples and full API reference information.
```

Comment:
This leads to a 404 error. We should leave it as is @derberg any thought about this?

Reply:
I do not know how/why link was incorrect, but I have accepted suggestion so now reference is working again

Reply:
@Florence-Njeri actually both are good - not sure where you see 404

I personally prefer full relative links like `/docs/tools/generator/api` but `api` works as well. Let's leave it as `api` as my approach for full relative links is an old habit that I follow but brainfart every time I try to find in my brain the reason for doing it 😄